components/forms/FormFieldRenderer.tsx

```
@@ -0,0 +1,170 @@
+            disabled={isFieldDisabled}
+            rows={4}
+            className="shadow-sm border-2 border-gray-200 focus:border-blue-500 focus:shadow-md transition-all duration-200"
+          />
+        );
+
+      case 'select':
+        const options =
+          field.selectOptions
+            ?.split(',')
+            .map((option) => option.trim())
+            .filter(Boolean) || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)